src/librustdoc/html/static/rustdoc.css

```diff
@@ -906,6 +906,10 @@ span.since {
 	.content .impl-items .method, .content .impl-items > .type, .impl-items > .associatedconstant {
 		display: flex;
 	}
+
+	.anchor {
```

Comment:
There isn't supposed to have an anchor in mobile version normally.

Reply:
But it appears when I click at the impl in Chromium's mobile view.

It can be removed with  `display:none !important`, then the `[+]` may be moved further right.

Reply:
I approve (for the mobile version)!